### ISSUE TITLE
CS4ZP6 Tester Feedback: Upload Button clickable when not editing

### ISSUE THREAD

**Issue by elwazana** [trevorNgo/Measure2.0#208]
Enviroment
- Google Chrome
- Windows 10

Describe the feedback/issue.
Once a rubric is marked as complete and editing is disabled, the file upload button in the learning outcomes are still clicable:
![image](https://user-images.githubusercontent.com/35268928/110222289-20cae600-7e9f-11eb-83b9-c502e5a5f1fd.png)

Priority
LOW

Please list the reproduction steps
- Login as `Department Coordinator && Instructor`
- Navigate to `Input Measurement`
- Open a learning outcome when editing is disabled (when `Mark As Complete` at the top is clicked)
- File upload button is still clickable

**Comment by trevorNgo** [comment]
but the file browser does not pop up right?

**Comment by elwazana** [comment]
No, but the text fields become non-clickable and so do the other buttons (save/delete). It just seems inappropriate for the file buttons to still be clickable even if they are non-functioning. It's why I put the priority as low.
